ci-operator/config/netobserv/network-observability-console-plugin/netobserv-network-observability-console-plugin-main.yaml

```
@@ -0,0 +1,133 @@
+base_images:
+  go-builder-base:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  web-builder-base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  use_build_cache: true
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    go-builder-base:
+      as:
+      - docker.io/library/golang:1.21
+    web-builder-base:
+      as:
+      - docker.io/library/node:16-alpine
+  to: network-observability-console-plugin
+- build_args:
+  - name: BUILDSCRIPT
+    value: :standalone
+  dockerfile_literal: |
+    ARG BUILDSCRIPT=
+    ARG LDFLAGS=
+    ARG TARGETPLATFORM=linux/amd64
+    ARG BUILDPLATFORM=linux/amd64
+    FROM --platform=$BUILDPLATFORM docker.io/library/node:16-alpine as web-builder
```

```suggestion
    FROM docker.io/library/node:16-alpine as web-builder
```

BUILDPLATFORM and TARGETPLATFORM are not supported. can you try removing it and run the build. Ideally, we shouldn't have the need for this.